src/WarcraftLegacies.Source/Quests/Lordaeron/QuestKingArthas.cs

```diff
@@ -38,11 +42,11 @@ public sealed class QuestKingArthas : QuestData
 
     /// <inheritdoc/>
     protected override string RewardFlavour =>
-      "With the Lich King eliminated, the Kingdom of Lordaeron is free of its greatest threat. King Terenas Menethil proudly abdicates in favor of his son.";
+      "With a buffer outpost in Northrend, the Kingdom of Lordaeron is safer of its greatest threat. King Terenas Menethil proudly abdicates in favor of his son, now an experienced warrior.";
```

Comment:
This breaks the flavour of the quest. The description says that the only thing standing in the way of Arthas' coronation is the Scourge, but then you don't actually have to deal with the Scourge to complete it. Possibly needs a total rewrite.

Reply:
Allright, il try and think of something

Reply:
i tried something, you can probably word it better